src/command/graph/check.rs

```diff
@@ -111,8 +111,10 @@ fn print_changes(
     let mut num_failures = 0;
 
     if !checks.is_empty() {
-        let mut table = Table::new();
-        table.add_row(row!["Change", "Code", "Description"]);
+        let mut table = table::get_table();
+
+        // bc => sets top row to be bold and center
```

Comment:
The added comments are perfect :) I imagine most of the time this will be copy/pasted anyway!